account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+"""Extend account.analytic.account."""
```

Comment:
MQT says we should follow pep standards, to be checked with pylint. Pep-0257 says "All modules should normally have docstrings", and pylint gives a missing-docstring error when it is not present. But then pylint will not complain when run in travis, because in the OCA setup only a small subset of checks has been enabled. I think here: travis/cfg/travis_run_pylint.cfg